helm/vitess/values.yaml

```diff
@@ -385,3 +388,12 @@ orchestrator:
     requests:
       cpu: 50m
       memory: 350Mi
+
+logtail:
+  image: vitess/logtail
+  tag: helm-1.0.6
```

Comment:
This is an older tag than what was being used earlier (1.0.7-5). Is there a reason why?
I'm also curious why we are publishing logtail, logrotate etc. in the first place. @dkhenry @derekperkins any insights?

Reply:
@deepthi i commited by mistake it's just example for testing

Reply:
@deepthi I wrote them because I couldn't find (and I spent a decent amount of time looking) any published artifacts that handled log files how we wanted to. I'm not attached to them if there's a better alternative that has popped up in the last 2 years.